recipes/perl-sys-cpu/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# This file created by conda-build 3.0.15+197.g41d59b5
+# meta.yaml template originally from:
+# /home/rdonnelly/conda/staged-recipes/recipes/perl-sys-cpu/0.61, last modified Sat Nov 11 23:55:48 2017
+# ------------------------------------------------
+#downgrade for perl 5.22 june 4th 2018
+
+package:
+    name: perl-sys-cpu
+    version: '0.61'
+source:
+    fn: Sys-CPU-0.61.tar.gz
+    sha256: 250a86b79c231001c4ae71d2f66428092a4fbb2070971acafd471aa49739c9e4
+    url: https://cpan.metacpan.org/authors/id/M/MZ/MZSANFORD/Sys-CPU-0.61.tar.gz
+build:
+    number: '0'
+    string: pl522_0
+    skip: true  # [win]  
+    skip: true  # [osx]
+requirements:
+    host:
+        - binutils_impl_linux-64 2.28.1 h04c84fa_2
+        - binutils_linux-64 7.2.0 hc67e822_10
+        - gcc_impl_linux-64 7.2.0 hc5ce805_2
+        - gcc_linux-64 7.2.0 h196ecd4_10
+        - libgcc-ng 7.2.0 h7cc24e2_2
```

Comment:
hard-coding the compiler like this is not advisable.  You should be using the compiler jinja2 expressions.  Please read https://conda.io/docs/user-guide/tasks/build-packages/compiler-tools.html#anaconda-compilers-and-conda-build-3

Reply:
ok thanks, i have take an existing recipe in anaconda.org

Reply:
It's done can we merge now ?